plantuml/LinkLoss/connection.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+skinparam defaultTextAlignment center
+
+|App|
+
+|Link Loss Service|
+
+|Gap|
+start
+
+|Link Loss Service|
+:connection;
+-> event;
+
+if (event.getStatus() == BLE_ERROR_NONE ?) then (yes)
+    :stop_alert()|
```

Comment:
shouldn't this check if alert is actually happening first?

Reply:
That check is performed in `stop_alert`. Please look at the sequence diagram for the function: [stop_alert.png](https://github.com/noonfom/mbed-os-experimental-ble-services/blob/seq-diag/plantuml/LinkLoss/img/stop_alert.png) or read the PlantUML: [stop_alert.puml](https://github.com/noonfom/mbed-os-experimental-ble-services/blob/seq-diag/plantuml/LinkLoss/stop_alert.puml). This is __not__ present in the implementation file: [LinkLossService.cpp](https://github.com/noonfom/mbed-os-experimental-ble-services/blob/ebf327d801610bb3261edd0521f43b7cdf0bc459/services/LinkLoss/source/LinkLossService.cpp#L81).

Reply:
OK, I was thinking in terms or roles and actions, this is following function implementations